src/include/distributed/multi_client_executor.h

```diff
@@ -72,6 +75,28 @@ typedef enum
 } BatchQueryStatus;
 
 
+/* Enumeration to track what the task associated with a connection is waiting for */
+typedef enum WaitStatus
+{
+	WAIT_BUSY, /* no IO required, ready to be processed further */
```

Comment:
On 2016-05-27 10:00:26 -0700, Jason Petersen wrote:
> I'd be remiss not to point out that (a) house style is to reserve the
> first enumeration for an explicit "invalid" value, and (b) house style
> is to explicitly assign each enum. There are arguments on either side
> of this, but I'd rather have a consistent codebase we can swap to
> another style than have a codebase whose style varies.

Gah, these are *bad* policies, especially the second one.